scripts/can_errgo/error_tui.go

```diff
@@ -0,0 +1,668 @@
+package main
+
+import (
+	"encoding/binary"
+	"flag"
+	"fmt"
+	"log"
+	"math"
+	"net"
+	"os"
+	"sort"
+	"strconv"
+	"time"
+
+	box "main/components/box"
+	table "main/components/table"
+
+	"github.com/charmbracelet/bubbles/key"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"golang.org/x/sys/unix"
+)
+
+
+const (
+	canInterface = "vcan0"  // Use the virtual CAN interface
+	canFrameSize = 16       // CAN frame size (for a standard CAN frame)
```

Comment:
why is this 16? Can frames have 8 data bytes

Reply:
Although the data frames is only 8 bytes the vcan has the structure:

4 bytes for Identifier
1 byte for DLC
1 byte padding
2 byte reserved
8 byte payload

I assume we will need the other bytes for information (Error name and description)

Reply:
ok thanks for the explanation